src/content/peerconnection/endtoend-encryption/index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <!--
- *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *  Copyright (c) 2020 The WebRTC project authors. All Rights Reserved.
```

Comment:
this one is changed enough that it's reasonable to call it new, even though it's also a branch from another file.